sips/sip-03x/sip-03x-btc-addresses-for-stacks-transactions.md

```diff
@@ -0,0 +1,215 @@
+
+## **Related Work**
+
+### **Bitcoin Address Formats**
+
+This proposal leverages existing Bitcoin address standards, specifically P2PKH (Pay-to-Public-Key-Hash) and P2WPKH (Pay-to-Witness-Public-Key-Hash).
+
+### **Lightning Network**
+
+The Lightning Network uses its own invoice format (BOLT11), which is different from on-chain Bitcoin addresses. This SIP aims to avoid introducing a new, distinct standard for sBTC addresses, learning from the friction this can cause.
+
+### **Multi-network Asset Support on Exchanges**
+
+Many cryptocurrency exchanges already support transferring the same asset (e.g., USDT, USDC) across multiple blockchain networks, often using the native address format of each respective network. This demonstrates a precedent for users managing assets at the same conceptual "address" (derived from the same key) but on different networks. This SIP proposes a similar user experience for Bitcoin and sBTC.
+
+## **Backwards Compatibility**
+
+This proposal is designed to be backwards compatible at the consensus layer, as no changes to the Stacks blockchain protocol itself are required.
+
+* Existing Stacks addresses will continue to function.  
+* The changes primarily affect client-side software (wallets, libraries, dApps).  
+* Applications can choose to implement support for Bitcoin addresses gradually.  
+* Users who prefer to use the existing Stacks address format can continue to do so.  
+* Smart contracts and on-chain logic will continue to use the underlying Stacks address format. The conversion to/from Bitcoin addresses will happen at the application/library layer before interacting with contracts.
+
+## **Activation**
+
+This SIP will be considered activated once the following conditions are met:
+
+1. **Updated Libraries**: Key Stacks developer libraries (e.g., Stacks.js or its successors) are updated to:  
+   * Accept P2PKH and P2WPKH Bitcoin addresses as input for transaction recipients.  
+   * Provide utility functions to convert Stacks addresses to P2WPKH Bitcoin addresses for display purposes.  
+   * Provide utility functions to resolve P2PKH/P2WPKH Bitcoin addresses back to Stacks addresses.  
+2. **Wallet Adoption**: At least two major Stacks wallets have implemented support for:  
+   * Displaying user account addresses in P2WPKH Bitcoin format.  
+   * Allowing users to send sBTC (and other Stacks assets) to P2PKH and P2WPKH Bitcoin addresses.  
+   * Implementing a warning mechanism (as described in the Specification section or similar) when sending sBTC to a Bitcoin address if sBTC support by the recipient cannot be confirmed.  
+3. **Explorer Adoption**: At least one major Stacks block explorer is updated to:  
```

Comment:
Is there more than one "major" Stacks block explorer besides the one Hiro runs?  Also, can the set of "major" explorers be enumerated?